src/vm/codeman.cpp

```diff
@@ -3293,6 +3392,29 @@ void EEJitManager::Unload(LoaderAllocator *pAllocator)
             break;
         }
     }
+#if defined(FEATURE_JIT_DROPPING)
+    ppList = m_JitDroppedDomainCodeHeaps.Table();
+    count = m_JitDroppedDomainCodeHeaps.Count();
+
+    for (int i=0; i < count; i++) {
+        if (ppList[i]->m_pAllocator== pAllocator) {
+            DomainCodeHeapList *pList = ppList[i];
+            m_JitDroppedDomainCodeHeaps.DeleteByIndex(i);
```

Comment:
I see it was the same as for m_DynamicDomainCodeHeaps in the original code...
But is that correct to remove in this way `m_JitDroppedDomainCodeHeaps.DeleteByIndex(i);`
inside a for loop iterating `for (int i=0; i < count; i++)`

Isn't the indexing of `m_JitDroppedDomainCodeHeaps` changing?